test/benchmark/instances.yaml

```diff
@@ -0,0 +1,4 @@
+test_vgg16_benchmark:
+  inf1.6xlarge:
+    instance_id: i-04b6adea9c066ad0f
+    key_filename: /Users/nikhilsk/nskool/serve/test/benchmark/ec2-key-name-6028.pem
```

Comment:
Can we the keys from a common shared location and not have user specific items here?

Reply:
Removed, and this file has been added to .gitignore.